CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # TigerBeetle Changelog
 
+## 2024-08-26
+
```

Comment:
Maybe add a note referencing https://github.com/tigerbeetle/tigerbeetle/issues/2231, that people can subscribe to that issue to be notified about breaking client stuff?